packages/shared/src/components/comments/AdAsComment.tsx

```diff
@@ -81,6 +81,7 @@ export const AdAsComment = ({ postId }: AdAsCommentProps): ReactElement => {
         size={ProfileImageSize.Large}
         user={{ id: providerId, username: source, image }}
         fallbackSrc={cloudinary.post.imageCoverPlaceholder}
+        style={{ backgroundColor: data?.backgroundColor }}
```

Comment:
Its only something we want for comment ads?

Reply:
this is the only comment we got from carbon so i guess yes 😓